README.md

```diff
@@ -1,2 +1,32 @@
 # TOL Api
 A node.js API client for the TOL APIs.
+
+###Requirements
+The api client requires [q](https://github.com/kriskowal/q), [request](https://github.com/request/request), and [underscore.js](http://underscorejs.org/).
+
+###Installation
+There are 2 ways you can import this project.
+
+1.) ```npm install tol-api```
+
+
+2.) Add the following entry to your packages.json and then run ```npm update``` (Recommended)
+
+```json
+"dependencies" :{
+        "tol-api" : "https://github.com/dominionenterprises.com/tol-api-nodejs#v0.7.x"
```

Comment:
Is there a reason for having the full url reference in the version rather than just the version number?